### ISSUE TITLE
Register if info_hash have been seen 

### ISSUE THREAD

**Issue by ad-m** [boramalper/magnetico#67]
Hello,

I think we should register when ``info_hash`` was seen even if no redownload them. We need info when info_hash has been available to track old ```info_hash``` without any node. Without monitoring such information we will find a database full of junk that no one is able to download ever again. Magnets sometimes expire, especially those less popular.

Greetings,

**Comment by ngosang** [comment]
We should save all metadata and add a "download button" in magneticow.

**Comment by ad-m** [comment]
@ngosang , i don't think this will help. If nobody can share metadata nobody can  share files, so metadata is useless.

**Comment by richardkiss** [comment]
I agree with @ad-m , magnet hashes are enough.

**Comment by ad-m** [comment]
It is also worth mention that saving all metadata would result in significant disk usage. If average metadata size is 0,3 MB and the speed is 6 metadata / second. The disk consumption increase is ~1,8 MB / second or 108MB / minute or 6,4 GB / hour or 153 GB / day. It goes beyond a large number of virtual servers shortly. True, such data can easily be stored using object storage, which is low in price. However, this seems pointless.
